src/index.html

```diff
@@ -17,16 +17,46 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-bar__container">
+      <form
+        class="search-bar search-bar_top"
+        data-qa="big"
+      >
+        <button class="search-bar__button">
+          <img
+            src="./images/Search.svg"
```

Comment:
Ensure that the `src` attribute points to the correct path of the image file. Verify that './images/Search.svg' is the correct path.